tests/parsers/moldev_softmax_pro/testdata/expected_abs_endpoint.json

```diff
@@ -1,5484 +1,5484 @@
 {
-    "$asm.manifest": "http://purl.allotrope.org/manifests/ultraviolet-absorbance/BENCHLING/2023/09/ultraviolet-absorbance.manifest",
     "measurement aggregate document": {
-      "measurement identifier": "7c29a45f-26b7-47ac-a3c8-918d23df8df1",
-      "plate well count": {
-        "value": 96,
-        "unit": "#"
-      },
-      "measurement document": [
-        {
-          "device control aggregate document": {
-            "device control document": [
-              {}
-            ]
-          },
-          "sample document": {
-            "well location identifier": "A1",
-            "plate barcode": "Plate1"
-          },
-          "compartment temperature": {
-            "value": 35.0,
-            "unit": "degC"
-          },
-          "data cube": {
-            "cube-structure": {
-              "dimensions": [
-                {
-                  "@componentDatatype": "int",
-                  "concept": "wavelength",
-                  "unit": "nm"
-                }
-              ],
-              "measures": [
-                {
-                  "@componentDatatype": "double",
-                  "concept": "absorbance",
-                  "unit": "mAU"
-                }
-              ]
-            },
-            "data": {
-              "dimensions": [
-                [
-                  426,
-                  426,
-                  426
-                ]
-              ],
-              "measures": [
-                [
-                  0.01,
-                  0.02,
-                  0.03
-                ]
-              ]
-            }
-          },
-          "processed data aggregate document": {
-            "processed data document": [
-              {
-                "data processing description": "processed data",
-                "processed data": 0.01
-              }
-            ]
-          }
+        "measurement identifier": "69b2d0aa-2a71-4998-b962-c95f9790f0d9",
```

Comment:
If this is just a refactor, then why did the expected test files change?

Reply:
@nathan-stender @bdworth After looking at the diffs of test files and running the unit tests on the old files this is what I found:

- `expected_lum_spectrum.json`: no significant changes on the file, rolled back to the previous version, all tests passed.
- `expected_abs_endpoint.json`: the type of the `dimensions` arrays items of the `data cube`s  changed from `int` to `str`. The `TDatacubeData` class defines the `dimenisions` property as a list of `TDimensionArray`, which accepts both `int` and `str`.
- `expected_fl_kinetic.json`: The problem here was that the `processed data` was being recorded with less significant figures when compared to the source file (ex. `0.014304468253968255` vs `0.0143044682539682`). It seems like the changes fixed this issue, but I haven't identified what was causing it.

Do you think these are good reasons to accept the changes to the test files?

Reply:
For `expected_abs_endpoint.json`, I think we should be getting the values back to `int`, the data cube values are permissive, but it's clear this data is really int from context.

For `expected_fl_kinetic.json`, if I read correctly we are increasing significant figures? (can't see the diff in github). If that is the case, I think this is OK

Reply:
@nathan-stender @slopez-b, I would hesitant to too caught up on the change data cube in `expected_abs_endpoint.json` because the in the planned update to the Plate Reader ASM, this data cube will be removed in the endpoint read.

Reply:
Just pushed a fix

Reply:
For `expected_fl_kinetic.json`, the current implementation is dropping digits compared to the source data. I don't think this is acceptable, as this is scientific data.

Reply:
Found an fixed

Reply:
@nathan-stender You are absolutely right. If mistakenly interpreted it the wrong way (I thought we were actually adding digits)